src/main/java/frc/robot/subsystems/shooter/ShooterSubsystem.java

```diff
@@ -0,0 +1,113 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.CoastOut;
+import com.ctre.phoenix6.controls.MotionMagicVelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterSubsystem extends SubsystemBase {
+    private final TalonFX leftMotor;
+    private final TalonFX rightMotor;
+
+    private final double kS = 0.0;
+    private final double kV = 0.0;
+    private final double kA = 0.0;
+
+    private final double kP = 0.0;
+    private final double kI = 0.0;
+    private final double kD = 0.0;
+
+    private final MotionMagicVelocityVoltage LEFT_MOTION_MAGIC_VELOCITY_VOLTAGE = new MotionMagicVelocityVoltage(0, 0,
+            false, 0, 0, false, false, false);
+    private final MotionMagicVelocityVoltage RIGHT_MOTION_MAGIC_VELOCITY_VOLTAGE = new MotionMagicVelocityVoltage(0, 0,
+            false, 0, 0, false, false, false);
+
+    // value is in rotations per second
+    // TODO: TUNE THIS TO A REASONABLE VALUE
+    private final double VEOLOCITY_TOLERANCE = 0;
+
+    public ShooterSubsystem() {
+        // TODO: find can ids
+        leftMotor = new TalonFX(0);
+        rightMotor = new TalonFX(0);
+
+        TalonFXConfiguration TALONFX_CONFIG = new TalonFXConfiguration();
+
+        // TODO: Check On Robot
+        TALONFX_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+
+        TALONFX_CONFIG.Slot0.kS = kS;
+        TALONFX_CONFIG.Slot0.kV = kV;
+        TALONFX_CONFIG.Slot0.kA = kA;
+
+        TALONFX_CONFIG.Slot0.kP = kP;
+        TALONFX_CONFIG.Slot0.kI = kI;
+        TALONFX_CONFIG.Slot0.kD = kD;
+
+        // TODO: Check On Robot
+
+        leftMotor.getConfigurator().apply(TALONFX_CONFIG);
+        rightMotor.getConfigurator().apply(TALONFX_CONFIG);
+
+        leftMotor.setInverted(false);
+        rightMotor.setInverted(false);
+    }
+
+    /**
+     * sets the target velocity for the left motor in rotations per second
+     * 
+     * @param velocity in rotations per second
+     */
+    public void setLeftVelocity(double velocity) {
+        LEFT_MOTION_MAGIC_VELOCITY_VOLTAGE.Velocity = velocity;
```

Comment:
We usually reserve CONSTANT_NAMES for things which are constant, that never change. Yet this one changes. Be aware that readers will assume it is constant by the way you've named it, and could make changes assuming that. Consider naming it differently.

Reply:
ok, will change to lower camel case